### PR TITLE
#3963 Adds new product type

### PR DESCRIPTION
**Pull request by Moejoe90**
This should wait for the backend implementation

### CODE REVIEW

File: packages/web/intl/locales/en-en.json [hunk 1]
```diff
@@ -421,6 +421,7 @@
   "eto.form.section.eto-terms.product.name.mini-eto-li": "Mini ETO in Liechtenstein",
   "eto.form.section.eto-terms.product.name.retail-eto-de": "Retail ETO in Germany",
   "eto.form.section.eto-terms.product.name.retail-eto-li-security": "Full Retail ETO in Liechtenstein",
+  "eto.form.section.eto-terms.product.name.retail-eto-li-sme": "Retail Growth ETO in Liechtenstein",
```
Comment (Moejoe90) [inline packages/web/intl/locales/en-en.json]: Confirm Product name

Comment (Moejoe90) [reply packages/web/intl/locales/en-en.json]: Confirmed with @pstroinski and @Vanilizer

File: packages/web/app/lib/api/eto/EtoProductsApi.interfaces.ts [hunk 2]
```diff
@@ -28,6 +28,7 @@ export enum EProductName {
   RETAIL_ETO_LI_SECURITY = "retail eto li security",
   RETAIL_ETO_LI_VMA = "retail eto li vma",
   FIFTH_FORCE_ETO = "hnwi eto de security",
+  RETAIL_EU_SME_ETO_LI_SECURITY = "retail EU-SME eto li security",
```
Comment (Moejoe90) [inline packages/web/app/lib/api/eto/EtoProductsApi.interfaces.ts]: Missing a confirmation for the product name

Comment (rudolfix) [reply packages/web/app/lib/api/eto/EtoProductsApi.interfaces.ts]: yes - this will be code name you get in /product from api

**Comment by Moejoe90** [comment]
I'm merging this without QA as it still waits for the backend implementation. 
In all cases everything should work

**Pull request merged**